content/docs/command-reference/freeze.md

```diff
@@ -1,32 +1,32 @@
 # freeze
 
-Freeze a [stage](/doc/command-reference/run). Use `dvc unfreeze` to unfreeze the
-stage.
+Fix the status of one or more [stages](/doc/command-reference/run) to _not
+changed_ until `dvc unfreeze` is used on it.
 
 ## Synopsis
 
 ```usage
 usage: dvc freeze [-h] [-q | -v] targets [targets ...]
 
 positional arguments:
-  targets        stages to freeze.
+  targets        Stages or .dvc files to freeze
```

Comment:
@pared please notice these updates which should probably be applied to the help output (dvc freeze -h) in the core repo.

Reply:
Similar changes to unfreeze.

Reply:
> p.s. there's an issue both in core and here to sync all these at some point.